exchange/bitswap/bitswap.go

```diff
@@ -442,10 +422,8 @@ func (bs *Bitswap) ReceiveError(err error) {
 // send strives to ensure that accounting is always performed when a message is
 // sent
 func (bs *Bitswap) send(ctx context.Context, p peer.ID, m bsmsg.BitSwapMessage) error {
-	defer log.EventBegin(ctx, "sendMessage", p, m).Done()
-	if err := bs.network.SendMessage(ctx, p, m); err != nil {
-		return err
-	}
+	//defer log.EventBegin(ctx, "sendMessage", p, m).Done()
```

Comment:
interesting, marshaling you were saying?

also btw, the events shouldn't marshal anything until they get written out-- meaning that removing their writing to disk should remove the perf hit.